### PR TITLE
SLVSCODE-984 adjust wording about free editions

### PR DESCRIPTION
**Pull request by sophio-japharidze-sonarsource**
[SLVSCODE-984](https://sonarsource.atlassian.net/browse/SLVSCODE-984)

[SLVSCODE-984]: https://sonarsource.atlassian.net/browse/SLVSCODE-984?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/lsp/server.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export function languageServerCommand(
 
   const params = [];
   if (DEBUG) {
-    params.push('-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8001,quiet=y');
+    params.push('-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8000,quiet=y');
```
Comment (kirill-knize-sonarsource) [inline src/lsp/server.ts:27]: Why change the port?

Comment (sophio-japharidze-sonarsource) [reply src/lsp/server.ts:27]: I accidentally changed the port to 8001 in a [previous commit](https://github.com/SonarSource/sonarlint-vscode/commit/ac87f1a74ee2ce75f36e374b76bc3d75785d3bdb#diff-871030d68031ea7ae229b410ff486fee8a1b1e637d56924d6376d1747bb87a72). This is just to revert the change

Comment (kirill-knize-sonarsource) [reply src/lsp/server.ts:27]: It's a change not related to the ticket. Can you please put it as a separate commit into this PR and don't squash them on merge? So history would be clear.

**Review by kirill-knize-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**